wizard/src/utils/state_mappings.json

```diff
@@ -0,0 +1,446 @@
+[{
+  "id": "geoId/01",
+  "name": "Alabama",
+  "iso_code": "US-AL",
+  "fips52AlphaCode": "AL"
```

Comment:
Catching up on PRs:

(1) Should we also resolve by geoId ? (aka FIPS) This is also very common.

(2) We today do not support resolution by fips52AlphaCode.  We might need to add it to:
- http://google3/datacommons/import/mcf_vocab.h;l=296;rcl=460356614
- https://github.com/datacommonsorg/import/blob/master/util/src/main/java/org/datacommons/util/Vocabulary.java#L268
- https://github.com/datacommonsorg/mixer/blob/master/internal/server/recon/resolve.go#L37
(unfortunately, these are 3 different binaries...)

Reply:
(1). Do you mean these as FIPS codes: https://www.nrcs.usda.gov/wps/portal/nrcs/detail/?cid=nrcs143_013696 If so, I wonder if this might dupe the detection more often than not because I can imagine numbers 01-79 appearing the data values column too.

(2). Interesting. I also don't see just FIPS codes in there. Would you also suggest adding those? Alternatively, do we have another property which refers to the 2-letter state codes?